src/views/CreateAlgorithm.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <main v-if="formState === 0" @keydown.enter="nextStep">
+    <ViewHeader title="How should the new algorithm be called?" :backLink="{ link: { name: 'algorithms' } }"/>
+    <Input :value.sync="newAlgorithm.name" name="Name" placeholder="Vehicle Routing for Package Delivery Algorithm" :autofocus="true"/>
+    <Button text="Continue" :onClick="nextStep"/>
+  </main>
+
+  <main v-else-if="formState === 1">
+    <ViewHeader
+      title="What information does the algorithm need?"
+      :backLink="{ onClick: previousStep }"
+    />
+    <ListSection title="Algorithm Inputs">
+      <li class="options slot input" v-for="resourceType in selectableResourceTypes" :key="resourceType.id" @click="toggleInputSelection(resourceType)">
+        <div class="checkbox" :class="{checked: resourceType.selected}" />
+        <div>
+          <p class="first-value">{{resourceType.name}}</p>
+          <p class="second-value">{{resourceType.parentType ? resourceType.parentType.name : ''}}</p>
```

Comment:
I would add the string "Parent type: " before the actual type to make clear why this name is there.

Reply:
Done